everycred/README.md

```diff
@@ -0,0 +1,20 @@
+# EveryCRED Verifiable Credentials
+
+### About:
+
+EVERYCRED is an exceptionally adaptable platform for the issuance, viewing, and verification of credentials. Notably, this platform is structured around the fundamental concepts of DID (Decentralized Identifiers) and W3C Verifiable Credentials, embracing cutting-edge technology and gametics.
```

Comment:
I am unable to find a likely-applicable definition of "gametics" anywhere, including the EveryCRED website. I suggest you either define "gametics" here, or change this word to something more easily understandable.